src/redis-benchmark.c

```diff
@@ -1787,11 +1787,6 @@ int main(int argc, char **argv) {
             }
             exit(1);
         }
-        if (config.cluster_node_count <= 1) {
-            fprintf(stderr, "Invalid cluster: %d node(s).\n",
-                    config.cluster_node_count);
-            exit(1);
```

Comment:
Can it be zero? What happens then?

Reply:
Could you think of any scenario? I tried reaching to a unavailable node and it fails with connection refused.

Reply:
That would be if CLUSTER NODES returns an empty set of nodes, or only replicas, or only masters without slots. Perhaps it is possible from a node in cluster mode that didn't MEET any other node. I'm not sure.

Maybe add `assert(config.cluster_node_count > 0);` just to be safe?

Reply:
Why not just update it to be == 0